docs/modules/ROOT/pages/vshn-managed/postgresql/plans.adoc

```diff
@@ -0,0 +1,155 @@
+= Plans and Sizing
+
+== Plans
+
+We provide a few preconfigured plans that should make sizing your PostgreSQL instance easier.
+You can just set one of the provided plans and your instance should run reliably.
+
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNPostgreSQL
+metadata:
+  name: example
+spec:
+  parameters:
+    size:
+      plan: standard-2
+  writeConnectionSecretToRef:
+    name: postgres-creds
+----
+
+By default, the following plans are available on every cluster:
+
+[cols="25a,15,15,15", options="header"]
+|===
+| Plan | CPU | Memory | Disk 
+| standard-2 | 400m | 1728Mi | 20Gi
+| standard-4 | 900m | 3776Mi | 40Gi
+| standard-8 | 1900m | 7872Mi | 80Gi
+|===
+
+NOTE: The reason why for example `standard-4` does not actually provide `4Gi` of usable memory is because of the sidecar overhead for each PostgreSQL instance.
+See section <<_sidecar_overhead>>.
+
+Depending on which cluster you're working with, there might be some slight differences in available plans.
+You can see all available plans running:
+
+[source,bash]
+----
+$ kubectl explain vshnpostgresql.spec.parameters.size.plan
+
+KIND:     VSHNPostgreSQL
+VERSION:  vshn.appcat.vshn.io/v1
+
+FIELD:    plan <string>
+
+DESCRIPTION:
+     Plan is the name of the resource plan that defines the compute resources.
+     The following plans are available:
+
+     standard-2 - CPU: 400m; Memory: 1728Mi; Disk: 20Gi
+
+     standard-4 - CPU: 900m; Memory: 3776Mi; Disk: 40Gi
+
+     standard-8 - CPU: 1900m; Memory: 7872Mi; Disk: 80Gi
+----
+
+[WARNING,caption="APPUiO Cloud"]
+====
+On APPUiO Cloud there are two important difference for the plans:
+
+* There is a `plus-X` variant of each plan that will schedule the instance on a https://docs.appuio.cloud/user/references/node-classes.html[Plus Node].
+* The `standard-8` plan is not available as it requests more memory than the https://docs.appuio.cloud/user/references/default-quota.html[Default Quota] allows.
+ If you need a larger instance, please contact https://docs.appuio.cloud/user/contact.html[APPUiO Cloud support] and manually request more memory according to <<_custom_sizing>>.
+====
+
+
+== Custom Sizing
+
+While we provide some pre-configured plans, you are free to choose custom values for CPU, memory, and disk size.
+To do that simply set `spec.parameters.size.cpu`, `spec.parameters.size.memory`, or `spec.parameters.size.disk`.
```

Comment:
Can a customer choose also between flex or plus nodes here?

Reply:
No. This done through the `scheduling` field. But I think I'll need to write this a bit more clearly